### PR TITLE
Add docs section in Link/Navlink for `reloadDocument`

### PR DESCRIPTION
**Pull request by jasikpark**
I see this is documented above, but I didn't realize it was an option until I was reading the source and saw it in the type of the props

### CODE REVIEW

File: docs/components/link.md [hunk 1]
```diff
@@ -147,3 +147,7 @@ let { state } = useLocation();
 [history-replace-state]: https://developer.mozilla.org/en-US/docs/Web/API/History/replaceState
 [history-push-state]: https://developer.mozilla.org/en-US/docs/Web/API/History/pushState
 [history-state]: https://developer.mozilla.org/en-US/docs/Web/API/History/state
+  
+## `reloadDocument`
+
+The `reloadDocument` property can be used to skip client side routing and let the browser handle the transition normally (as if it were an `<a href>`).
```
Comment (timdorr) [inline docs/components/link.md:153]: This needs to be moved up above the anchors.

File: docs/components/nav-link.md [hunk 2]
```diff
@@ -121,3 +121,7 @@ Adding the `caseSensitive` prop changes the matching logic to make it case sensi
 When a `NavLink` is active it will automatically apply `<a aria-current="page">` to the underlying anchor tag. See [aria-current][aria-current] on MDN.
 
 [aria-current]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current
+  
+## `reloadDocument`
+
+The `reloadDocument` property can be used to skip client side routing and let the browser handle the transition normally (as if it were an `<a href>`).
```
Comment (timdorr) [inline docs/components/nav-link.md:127]: Same with this one.

File: docs/components/link.md (hunk 1 above)
Comment (jasikpark) [reply docs/components/link.md:153]: what do you mean by "anchors"

Comment (jasikpark) [reply docs/components/link.md]: done!

**Comment by timdorr** [comment]
Thanks!

**Pull request merged**